x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/action/TransportDeleteEnrichPolicyAction.java

```diff
@@ -128,7 +129,12 @@ protected void masterOperation(
             return;
         }
 
-        deleteIndicesAndPolicy(request.getName(), ActionListener.wrap((response) -> {
+        GetIndexRequest indices = new GetIndexRequest().indices(EnrichPolicy.getBaseName(request.getName()) + "-*")
```

Comment:
Maybe add a comment here why the delete enrich index api can't rely on just deleting all the enrich indices by a wildcard index expression?

Reply:
OK, I have done that.

Reply:
thank you!